examples/example-with-encryption/README.md

```diff
@@ -0,0 +1,23 @@
+# Consul cluster with encryption example
+
+This folder contains an example of a Packer manifest that creates an AMI with the required certs for TLS validation, as well as installing an updated version of the `run-consul` script that accepts parameters for enabling RPC and/or gossip encryption.
+
+The resulting AMI id can then be passed as a parameter to `variables.tf`. The `enable_gossip_encryption` and `enable_rpc_encryption` variables are set to `true` by default in this example, but they don't have to be in your implementation. In this example they're passed as parameters to the `user_data` template to generate the flags passed to `run-consul` but you can use a different strategy.
+
+The end result of this example should be a cluster of 3 Consul servers and 3 Consul clients, all running on individual EC2 instances. If the default variables are used, both gossip and RPC encryption will be enabled. You can validate this by trying to bring up another Consul node or cluster NOT running with encryption and attempt to join the existing cluster.
+
+Running this example with encryption turned off and then attempt to upgrade it to use encryption is a good exercise to validate that a production cluster can be upgraded with minimal impact.
+
+To understand more about how Consul handles encryption or how you can upgrade to use encryption without downtime, check out the [Consul encryption documentation](https://www.consul.io/docs/agent/encryption.html).
```

Comment:
Please add a big BOLD warning that says the certs in this repo are JUST for learning/testing and should NOT be used in prod! I know the TLS certs folder's README says this, but we should have this warning in as many places as possible :)